modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -174,8 +174,11 @@ public Connection getConnection() {
     void updateBindings(List<Binding> bindings) throws SQLException {
 
         getConnection();
+        PreparedStatement preparedStatement = null;
+        ResultSet resultSet = null;
+
         try {
-            PreparedStatement preparedStatement = conn.prepareStatement(UPDATE_BINDING);
+            preparedStatement = conn.prepareStatement(UPDATE_BINDING);
```

Comment:
Can't we use try-with-resources[1] here? Then you don't need to specifically close the resources. 

[1] https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html